packages/babel-plugin-proposal-partial-application/test/fixtures/general/multiple-calls/options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": ["proposal-partial-application"]
+}
```

Comment:
nit: The `options.json` files are the same for all tests as I can see, so the config file could be moved on level up into the `general` folder. Then all the fixtures in `general` would share the config.

Reply:
@danez Thanks for the feedback. 
I moved `options.json` one level up and added a new test.